apps/meteor/tests/end-to-end/api/11-permissions.js

```diff
@@ -54,6 +56,18 @@ describe('[Permissions]', function () {
 	});
 
 	describe('[/permissions.update]', () => {
+		let testUser;
+		let testUserCredentials;
+		before(async () => {
+			testUser = await createUser();
+			testUserCredentials = await login(testUser.username, password);
+			await updatePermission('access-permissions', ['admin']);
```

Comment:
Let's just reset to default `after` all. I know the value will be the same but in case someone adds another test here changing this permission, we could mess up with the subsequent tests